requirements.txt

```diff
@@ -26,6 +26,7 @@ pytest-asyncio
 pytest-trio
 pytest-cov
 trio
+trio-typing
```

Comment:
Do we need to add this dependency?

Reply:
For any trio types to be type-checked this must be installed (the only type being checked is `trio.Event`). This package also has a mypy plugin that can be enabled for extra functionality, which I did enable in encode/httpcore#69, but I didn't bother here.